cloudify_agent/operations.py

```diff
@@ -508,6 +515,22 @@ def _create_broker_config(transfer_mode=False):
     ctx.instance.update()
 
 
+def _update_broker_config(agent, manager_ip, manager_cert):
```

Comment:
I think there's no point in passing the agent here. Just get it from the runtime properties